### PR TITLE
Correct .md files inconsistency and unclear content, fix hyper link.

### PR DESCRIPTION
**Pull request by wantedfast**
1.Correct all 'Key Vault' to 'Azure Key Vault'
2.Fix content hyper link.

### CODE REVIEW

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md [hunk 1]
```diff
@@ -52,21 +52,21 @@ Use the [Azure CLI][azure_cli] snippet below to create/get client secret credent
         "tenant": "tenant-ID"
     }
     ```
-* Use the returned credentials above to set  **AZURE_CLIENT_ID**(appId), **AZURE_CLIENT_SECRET**(password) and **AZURE_TENANT_ID**(tenant) environment variables. The following example shows a way to do this in Powershell:
+* Use the returned credentials above to set  **AZURE_CLIENT_ID** (appId), **AZURE_CLIENT_SECRET** (password) and **AZURE_TENANT_ID** (tenantId) environment variables. The following example shows a way to do this in Powershell:
```
Comment (jongio) [inline sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md:55]: oxford comma

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md [hunk 2]
```diff
@@ -52,21 +52,21 @@ Use the [Azure CLI][azure_cli] snippet below to create/get client secret credent
...
     ```PowerShell
     $Env:AZURE_CLIENT_ID="generated-app-ID"
     $Env:AZURE_CLIENT_SECRET="random-password"
     $Env:AZURE_TENANT_ID="tenant-ID"
     ```
 
-* Grant the above mentioned application authorization to perform key operations on the key vault:
+* Grant the above mentioned application authorization to perform certificate operations on the Azure key vault:
```
Comment (jongio) [inline sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md:62]: capital K and V

**Review by jongio — CHANGES_REQUESTED** [review]

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
+Azure Key Vault is a cloud service that provides secure storage and automated management of certificates used throughout a cloud application. Multiple certificates, and multiple versions of the same certificate, can be kept in the Key Vault. Each certificate in the vault has a policy associated with it which controls the issuance and lifetime of the certificate, along with actions to be taken as certificates near expiry.
 
-The Azure Key Vault Certificate client library enables programmatically managing certificates, offering methods to create, update, list, and delete certificates, policies, issuers, and contacts. The library also supports managing pending certificate operations and management of deleted certificates.
+The Azure Key Vault Certificates client library enables programmatically managing certificates, offering methods to create, update, list, and delete certificates, policies, issuers, and contacts. The library also supports managing pending certificate operations and management of deleted certificates.
```
Comment (heaths) [inline sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md:4]: "Certificates" is not a proper noun and should be lowercase.

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md [hunk 4]
```diff
@@ -229,9 +229,9 @@ await client.PurgeDeletedCertificateAsync(secret.Name);
 ## Troubleshooting
 
 ### General
-When you interact with the Azure Key Vault Certificate client library using the .NET SDK, errors returned by the service correspond to the same HTTP status codes returned for [REST API][keyvault_rest] requests.
+When you interact with the Azure Key Vault Certificates client library using the .NET SDK, errors returned by the service correspond to the same HTTP status codes returned for [REST API][keyvault_rest] requests.
```
Comment (heaths) [inline sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md:232]: Not a proper noun and should be lowercase.

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/samples/Sample2_GetCertificates.md [hunk 5]
```diff
@@ -115,4 +115,4 @@ To see the full example source, see:
 * [Synchronous Sample2_GetCertificates.cs](../tests/samples/Sample2_GetCertificates.cs)
 * [Asynchronous Sample2_GetCertificatesAsync.cs](../tests/samples/Sample2_GetCertificatesAsync.cs)
 
-[DefaultAzureCredential]: ../../../identity/Azure.Identity/README.md
+[DefaultAzureCredential]: ../../../identity/Azure.Identity/README.md/#DefaultAzureCredential
```
Comment (heaths) [inline sdk/keyvault/Azure.Security.KeyVault.Certificates/samples/Sample2_GetCertificates.md:118]: Same comment here as on other PRs.

**Review by heaths — CHANGES_REQUESTED** [review]
Same comments as on other PRs in sdk\keyvault (please combine these in the future, but thanks for taking the time to make them).

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md (hunk 1 above)
Comment (wantedfast) [reply sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md:55]: All comments resolved in New PR #11295

**Comment by heaths** [comment]
Should this be closed now?

**Pull request closed without merging**